src/main/resources/changelogs/0003-add-unique-constraint-to-cart.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="1_delete_duplicate_carts" author="raluca">
+        <sqlFile encoding="utf8" path="sql/delete-duplicate-carts-script.sql"/>
```

Comment:
there is a solution for this issue, please try it and if it works use it, instead of having a separate file:
<sql>
<![CDATA[
	--SQL CODE
  ]]>
<sql>

Reply:
it worked 💃